documents/src/pages/utils/cache.md

```diff
@@ -0,0 +1,133 @@
+<!-- 
+title: Local caching Utility
+location: ./utils/cache
+type: page
+layout: default
+-->
+
+::status-complete::
+
+# Local cache
+The utility provides provides an alternative way to store data on your local machine.
+There are 2 main classes that collaborate: wrapper and storage. LocalCache is high level class that wraps the selected storage to be asynchronous api.
+The second is storage class. Store data on the user's machine. The storages has a list of support as below:
+- localstorage
+- indexeddb
```

Comment:
We should mention suitable use cases for each storage option.

Reply:
Updated. I put more info of each storages.